prime-dotnet-webapi/Migrations/20200810222109_AgreementRefactor.cs

```diff
@@ -80,65 +80,93 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 nullable: false,
                 defaultValue: "");
 
+            // Rename and re-link Agreement
```

Comment:
**NOTE:** In order for the migration to run when we rebuild our DB locally the updates and inserts need to come after the table rename, otherwise it throws an error saying there is no entity type of `UserClause`.  This change only moves them after the rename and changes the name of the table to `Agreement` to reflect the renaming of the table.